concepts/Common/DevExtreme CLI/30 Migrate to Named Configuration Components.md

```diff
@@ -0,0 +1,46 @@
+DevExtreme Angular v24.2 introduced named configuration components. These are strictly typed and specific to each UI component. Strict typing allows code completion and hinting systems to suggest only valid configurations and detect invalid usage. We recommend upgrading to named configuration components to improve your development experience.
```

Comment:
I think we can even use smth like Note (but not in red color) for the text "We recommend upgrading to named configuration components to improve your development experience."

I think we can merge this in one note with "Starting with v25.2, a warning is displayed in the console if you use generic configuration components."